lighthouse-core/test/audits/dobetterweb/appcache-manifest-test.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+const Audit = require('../../../audits/dobetterweb/appcache-manifest.js');
```

Comment:
this isn't on you at all since almost all the audit tests do this, but do you want to start a trend in audit test of actually naming this variable after the audit? Makes the tests below easier to follow.

`const AppcacheManifestAudit =` or even just `const AppcacheManifest =` or whatever